acceptance/bundle/artifacts/whl_prebuilt_outside/this_dab/databricks.yml

```diff
@@ -0,0 +1,21 @@
+bundle:
+  name: python-wheel
+
+sync:
```

Comment:
Shouldn't you use this to make it work actually?
```
sync:
  paths:
    - ..
```

Reply:
Ah right, +1. include exclude section are only mean for the gitignore patterns.

Reply:
Thanks, updated here 56c37c7b8dfb5ba6832fab639ab3ea91cacd9fbb 

Still failing:

```
>>> errcode [CLI] bundle deploy
Uploading other_dab/dist/lib/other_test_code-0.0.1-py3-none-any.whl...
Uploading other_dab/dist/my_test_code-0.0.1-py3-none-any.whl...
Uploading bundle files to /Workspace/Users/[USERNAME]/.bundle/python-wheel/default/files...
Error: cannot list files: stat other_dab/dist/*.whl: no such file or directory
Error: stat other_dab/dist/*.whl: no such file or directory
```

Reply:
thanks 1045242b220f49c15768c7cf2bd6c281bbb92fe5